site/usage/applications.md

```diff
@@ -7,41 +7,53 @@ permalink: /applications.html
 -   [Vega-Lite for Julia](https://github.com/fredo-dedup/VegaLite.jl)
 -   [Vega-Lite "bindings" for R](https://github.com/hrbrmstr/vegalite), create Vega-Lite visualizations in R.
 -   [vegaliteR](https://github.com/timelyportfolio/vegaliteR), vega-lite htmlwidget for R.
 -   [Vegas](https://github.com/aishfenton/Vegas) brings visualizations to Scala and Spark using Vega-Lite.
 -   [Vizard](https://github.com/yieldbot/vizard) provides a tiny REPL client to visualize Clojure data in the browser.
 
-## Tools
+## Tools where you can use Vega-Lite
 
--   [Voyager (2)](https://github.com/vega/voyager), visualization tool for exploratory data analysis that blends a Tableau-style specification interface (formerly [Polestar](https://github.com/vega/polestar)) with chart recommendations (formerly the Voyager visualization browser).
--   [Lyra](https://github.com/vega/lyra), an interactive, graphical Visualization Design Environment (VDE)
--   [Vega-Editor](https://vega.github.io/editor/), the editor for Vega and Vega-Lite.
 -   [JupyterLab](https://github.com/jupyterlab/jupyterlab), an extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture.
--   [PdVega](https://jakevdp.github.io/pdvega/), lets you create interactive Vega-Lite plots for Pandas.
 -   [nteract](https://github.com/nteract/nteract), interactive notebook application with Vega and Vega-Lite renderer.
--   [Observable](https://beta.observablehq.com/), an interactive JavaScript notebook.
+-   <span class="octicon octicon-star"></span> [Observable](https://beta.observablehq.com/), an interactive JavaScript notebook. [Example](https://beta.observablehq.com/@mbostock/exploring-data-with-vega-lite).
+-   [data.world](https://data.world), upload `.vg.json` and `.vl.json` files along side your raw data, or [embed Vega](https://docs.data.world/tutorials/markdown/#vega-and-vega-lite) directly into comments and summary markdown.
+-   <span class="octicon octicon-star"></span> [vega-desktop](https://github.com/kristw/vega-desktop), a desktop app that let you open `.vg.json` and `.vl.json` to see visualizations just like you open image files with an image viewer. Can also be used for [creating visualizations with Vega/Vega-Lite locally](https://medium.com/@kristw/create-visualizations-with-vega-on-your-machine-using-your-preferred-editor-529e1be875c0).
+
+## Authoring Tools
+
+-   <span class="octicon octicon-star"></span> [Vega-Editor](https://vega.github.io/editor/), the editor for Vega and Vega-Lite.
+
+## Tools that use Vega-Lite
+
+-   <span class="octicon octicon-star"></span> [Voyager (2)](https://github.com/vega/voyager), visualization tool for exploratory data analysis that blends a Tableau-style specification interface (formerly [Polestar](https://github.com/vega/polestar)) with chart recommendations (formerly the Voyager visualization browser).
```

Comment:
Voyager is like a GUI for visual authoring Vega-Lite, so it's a bit different from the rest?  

I'd like to think that Vega-Desktop, Vega-Editor, Voyager are all tools for using Vega-Lite?